lib/css_parser/regexps.rb

```diff
@@ -1,65 +1,66 @@
   end
 
   # :stopdoc:
   # Base types
   RE_NL = Regexp.new('(\n|\r\n|\r|\f)')
-  RE_NON_ASCII = Regexp.new('([\x00-\xFF])', Regexp::IGNORECASE, 'n')  #[^\0-\177]
+  RE_NON_ASCII = Regexp.new('([\x00-\xFF])', Regexp::IGNORECASE, 'n') # [^\0-\177]
   RE_UNICODE = Regexp.new('(\\\\[0-9a-f]{1,6}(\r\n|[ \n\r\t\f])*)', Regexp::IGNORECASE | Regexp::EXTENDED | Regexp::MULTILINE, 'n')
   RE_ESCAPE = Regexp.union(RE_UNICODE, '|(\\\\[^\n\r\f0-9a-f])')
   RE_IDENT = Regexp.new("[\-]?([_a-z]|#{RE_NON_ASCII}|#{RE_ESCAPE})([_a-z0-9\-]|#{RE_NON_ASCII}|#{RE_ESCAPE})*", Regexp::IGNORECASE, 'n')
 
   # General strings
-  RE_STRING1 = Regexp.new('(\"(.[^\n\r\f\\"]*|\\\\' + RE_NL.to_s + '|' + RE_ESCAPE.to_s + ')*\")')
-  RE_STRING2 = Regexp.new('(\'(.[^\n\r\f\\\']*|\\\\' + RE_NL.to_s + '|' + RE_ESCAPE.to_s + ')*\')')
+  RE_STRING1 = Regexp.new("(\"(.[^\n\r\f\"]*|\\\\#{RE_NL}|#{RE_ESCAPE})*\")")
+  RE_STRING2 = Regexp.new("('(.[^\\n\\r\\f\\']*|\\\\#{RE_NL}|#{RE_ESCAPE})*')")
   RE_STRING = Regexp.union(RE_STRING1, RE_STRING2)
 
   RE_INHERIT = regex_possible_values 'inherit'
 
-  RE_URI = Regexp.new('(url\([\s]*([\s]*' + RE_STRING.to_s + '[\s]*)[\s]*\))|(url\([\s]*([!#$%&*\-~]|' + RE_NON_ASCII.to_s + '|' + RE_ESCAPE.to_s + ')*[\s]*)\)', Regexp::IGNORECASE | Regexp::EXTENDED  | Regexp::MULTILINE, 'n')
-  URI_RX = /url\(("([^"]*)"|'([^']*)'|([^)]*))\)/im
-  RE_GRADIENT = /[-a-z]*gradient\([-a-z0-9 .,#%()]*\)/im
+  RE_URI = Regexp.new("(url\\([\\s]*([\\s]*#{RE_STRING}[\\s]*)[\\s]*\\))|(url\\([\\s]*([!#$%&*\\-~]|#{RE_NON_ASCII}|#{RE_ESCAPE})*[\\s]*)\\)", Regexp::IGNORECASE | Regexp::EXTENDED | Regexp::MULTILINE, 'n')
+  URI_RX = /url\(("([^"]*)"|'([^']*)'|([^)]*))\)/im.freeze
+  RE_GRADIENT = /[-a-z]*gradient\([-a-z0-9 .,#%()]*\)/im.freeze
 
   # Initial parsing
-  RE_AT_IMPORT_RULE = /\@import\s+(url\()?["']?(.[^'"\s]*)["']?\)?([\w\s\,^\])]*)\)?;?/
+  RE_AT_IMPORT_RULE = /@import\s+(url\()?["']?(.[^'"\s]*)["']?\)?([\w\s,^\])]*)\)?;?/.freeze
 
   #--
-  #RE_AT_MEDIA_RULE = Regexp.new('(\"(.[^\n\r\f\\"]*|\\\\' + RE_NL.to_s + '|' + RE_ESCAPE.to_s + ')*\")')
+  # RE_AT_MEDIA_RULE = Regexp.new('(\"(.[^\n\r\f\\"]*|\\\\' + RE_NL.to_s + '|' + RE_ESCAPE.to_s + ')*\")')
 
-  #RE_AT_IMPORT_RULE = Regexp.new('@import[\s]*(' + RE_STRING.to_s + ')([\w\s\,]*)[;]?', Regexp::IGNORECASE) -- should handle url() even though it is not allowed
+  # RE_AT_IMPORT_RULE = Regexp.new('@import[\s]*(' + RE_STRING.to_s + ')([\w\s\,]*)[;]?', Regexp::IGNORECASE) -- should handle url() even though it is not allowed
   #++
-  IMPORTANT_IN_PROPERTY_RX = /[\s]*!important\b[\s]*/i
+  IMPORTANT_IN_PROPERTY_RX = /\s*!important\b\s*/i.freeze
 
   RE_INSIDE_OUTSIDE = regex_possible_values 'inside', 'outside'
   RE_SCROLL_FIXED = regex_possible_values 'scroll', 'fixed'
   RE_REPEAT = regex_possible_values 'repeat(\-x|\-y)*|no\-repeat'
   RE_LIST_STYLE_TYPE = regex_possible_values 'disc', 'circle', 'square', 'decimal-leading-zero', 'decimal', 'lower-roman',
```

Comment:
more uniform when all have 2-space
```suggestion
  RE_LIST_STYLE_TYPE = regex_possible_values(
    'disc', 'circle', 'square', 'decimal-leading-zero', 'decimal', 'lower-roman',
```